missions/ULM_missions.txt

```diff
@@ -0,0 +1,335 @@
+ULM_diplomacy = {
+	slot = 1
+	generic = no
+	ai = yes
+	potential = {
+		tag = ULM
+	}
+	has_country_shield = yes
+
+
+	ULM_hre_allies = {
+		icon = mission_alliances.png
```

Comment:
Does .png work? I thought you needed to create a .dds file (I use paint.net), then acknowledge its existence in a `interface/mymissions.gfx` file.

Reply:
fairly certain it shouldn't be .png (in my previous mods i've never used png) but pdx wiki told me so and i'm not good at coding syntax yet, so i figured fukkit. couldn't test it yet.

Reply:
The wiki (https://eu4.paradoxwikis.com/Mission_modding#Interface) mentions `.dds` files.

Reply:
then idk man >.>